src/window.rs

```diff
@@ -1231,6 +1231,16 @@ impl Window {
     pub fn set_cursor_hittest(&self, hittest: bool) -> Result<(), ExternalError> {
         self.window.set_cursor_hittest(hittest)
     }
+
+    /// Returns the current cursor position in screen coordinates.
```

Comment:
Perhaps it would make sense to convert it to window coordinates first, then?

Reply:
Idk, maybe, but the cursor could be out of the window, should it report (0,0) in that case?

Reply:
Maybe it should just return an `option`?

In any case, I think we should keep just the one on `EventLoopTarget`, that at least makes it very clear that the position is in screen-coordinates, and not window coordinates (and getting window-relative cursor position is fairly easy using `Window::inner_position`)

Reply:
removed it from `Window` as I couldn't think of an immediate use-case for it.